general/solomiia_tymoshchuk/lecture3/_2/BirthDayValidator.java

```diff
@@ -0,0 +1,24 @@
+package general.solomiia_tymoshchuk.lecture3._2;
+
+import static java.lang.Integer.parseInt;
+
+public class BirthDayValidator extends AccountValidator
```

Comment:
the task contained a list of validation rules, you needed to implement a class for a specific rule and not for a specific field (note: some rules can be applied to more than one field, e.g., not empty)